CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixed
 
+- Fix arrow color of language selector in footer
```

Comment:
You probably shouldn't add this line if the bug was introduced since the latest release...